dtfs-central-api/test-helpers/test-data/tfm-facility.ts

```diff
@@ -2,18 +2,30 @@ import { ObjectId } from 'mongodb';
 import { TfmFacility } from '@ukef/dtfs2-common';
 import { generateMockPortalUserAuditDatabaseRecord } from '@ukef/dtfs2-common/change-stream/test-helpers';
 import { aFacility } from './facility';
+import { KeyingSheetCalculationTfmFacilityValues } from '../../src/types/tfm/tfm-facility';
+
+const facility = aFacility();
 
 export const aTfmFacility = (): TfmFacility => {
   const tfmFacilityId = new ObjectId();
 
   return {
     _id: tfmFacilityId,
     facilitySnapshot: {
-      ...aFacility(),
+      ...facility,
       _id: tfmFacilityId,
     },
     amendments: [],
     tfm: {},
     auditRecord: generateMockPortalUserAuditDatabaseRecord(new ObjectId()),
   };
 };
+
+export const keyingSheetCalculationTfmFacilityValues = {
+  coverEndDate: new Date(),
+  coverStartDate: new Date(),
+  dayCountBasis: facility.dayCountBasis,
+  interestPercentage: facility.interestPercentage,
+  coverPercentage: facility.coverPercentage,
+  value: facility.value,
```

Comment:
Could destructure these properties on line 7 so instead of this:

```ts
const facility = aFacility();
```

Could be:

```ts
const { dayCountBasis, interestPercentage, coverPercentage, value } = aFacility();
```

Reply:
done, I still need the const facility = as it is used in aTfmFacility